index.html

```diff
@@ -528,6 +529,59 @@ <h2>Glossary</h2>
 </section>
 
 
+<section class="appendix" id="how-to-use">
+	<h3 id="howto">How to use this glossary</h3>
+	
+	<p>If you are writing a W3C specification, you can use the terminology in this document directly by importing this document as a cross-reference source. This may be especially helpful when using <a>character encoding</a> or <a>locale</a>-related terms found here. Such terms are common to the needs of many specifications and often refer to other definitions found in this glossary. In most cases, though, copying these definitions into your specification won't add value to your readers.</p>
+	
+    <p>This document is on the WG NOTE track in order to facilitate maintenance. This means that the definitions in the glossary are defined to be <em>informative</em> rather than <em>normative</em>.</p>
```

Comment:
Is it going to be a WG NOTE or a DRAFT NOTE?

I suspect that the way a definition is used by a spec determines whether or not that definition is informative or normative - not the qualities of our glossary.  That goes back to my previous comment.

Reply:
Our document is "on the WG NOTE track", so is either a NOTE or a DRAFT NOTE, depending on when you look at it (the TR version should probably always be a NOTE??). The point is that _officially_ it can't be a normative reference and the definitions in it when used by reference are informative. If someone needs a normative reference, they'll have to pull the definition out.

I know this has to do with your previous comment as well and I'll add text about that use case. I'll look at whether a rephrasing is needed here.